airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSourceOperations.java

```diff
@@ -211,6 +213,12 @@ private void putMoney(final ObjectNode node, final String columnName, final Resu
     node.put(columnName, DataTypeUtils.returnNullIfInvalid(() -> Double.valueOf(moneyValue), Double::isFinite));
   }
 
+  private void putHstoreAsJson(final ObjectNode node, final String columnName, final ResultSet resultSet, final int index)
+      throws SQLException {
+    final var data = resultSet.getObject(index);
+    node.put(columnName, new Gson().toJson(data));
```

Comment:
nitpick: most of our codebase uses ObjectMapper rather than Gson, e.g. https://github.com/airbytehq/airbyte/blob/da88158c2f253790fc26e2c68d538607afd61861/airbyte-integrations/connectors/source-sftp/src/main/java/io/airbyte/integrations/source/sftp/parsers/CsvFileParser.java#L20

Reply:
Fixed

Reply:
I think it would be better to declare the object mapper as a `static final` constant

Reply:
done